server/graphql/types/src/types/temperature_breach.rs

```diff
@@ -144,6 +144,15 @@ impl TemperatureBreachNode {
             .await?
             .map(LocationNode::from_domain))
     }
+
+    pub async fn temperature_log(&self, ctx: &Context<'_>) -> Result<Option<TemperatureLogNode>> {
```

Comment:
Needed this for the temperature...

Reply:
It seems like the design is to record the breach_id against all temperature log records related (so there might actually be a whole lot of associated temperature log records)

I'm guessing the design is intending to show either the max temperature or the min temperature (depending on the breach type?

![image](https://github.com/openmsupply/open-msupply/assets/8287373/0799e3df-0e8c-4905-9a68-c8c087513b71)

Reply:
yeah, my preference would be to return just the temperature.